sdk/servicebus/Azure.Messaging.ServiceBus/src/Azure.Messaging.ServiceBus.csproj

```diff
@@ -21,7 +21,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
     <PackageReference Include="Azure.Core.Amqp" />
-    <PackageReference Include="Microsoft.Azure.Amqp" />
+    <PackageReference Include="Microsoft.Azure.Amqp" VersionOverride="2.5.1" />
```

Comment:
We should update the packages.data.props file once we have verified the new version.

Reply:
If the Service Bus changes validate, moving this to the central version and then running the Event Hubs tests should work to validate;  we can continue investigating the other changes in 2.5.1 separately.